### PR TITLE
Resolves #2538

### LINKED ISSUE #2538: might be worth mentioning a restart isn't required...
...unlike disabling TLS 1.0 or TLS 1.1 or RC4, which does require a restart
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ca6ec136-cd94-c595-0078-46fed97745ed
* Version Independent ID: 0e9c9820-20e0-b240-0995-f29948f00b31
* Content: [Enable-TlsCipherSuite (TLS)](https://docs.microsoft.com/en-us/powershell/module/tls/enable-tlsciphersuite?view=windowsserver2019-ps)
* Content Source: [docset/winserver2019-ps/tls/Enable-TlsCipherSuite.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/winserver2019-ps/tls/Enable-TlsCipherSuite.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @JasonGerend
* Microsoft Alias: **jgerend**

### PR DESCRIPTION
**Pull request by velkovb**
Update cmdlet description
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2538

### CODE REVIEW

File: docset/winserver2016-ps/tls/Enable-TlsCipherSuite.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ Enable-TlsCipherSuite [[-Position] <UInt32>] [-Name] <String> [-WhatIf] [-Confir
 
 ## DESCRIPTION
 The **Enable-TlsCipherSuite** cmdlet enables a cipher suite.
-This cmdlet adds the cipher suite to the list of Transport Layer Security (TLS) protocol cipher suites for the computer.
-If you do not specify a position in the list, this cmdlet adds it at the lowest position.
+This cmdlet adds the cipher suite to the list of Transport Layer Security (TLS) protocol cipher suites for the computer. 
+If you do not specify a position in the list, this cmdlet adds it at the lowest position. No restart is required in order for changes to take effect.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/tls/Enable-TlsCipherSuite.md:25]:
```suggestion
If you do not specify a position in the list, this cmdlet adds it at the lowest position. No restart is required for changes to take effect.
```

File: docset/winserver2019-ps/tls/Enable-TlsCipherSuite.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ Enable-TlsCipherSuite [[-Position] <UInt32>] [-Name] <String> [-WhatIf] [-Confir
 ## DESCRIPTION
 The **Enable-TlsCipherSuite** cmdlet enables a cipher suite.
 This cmdlet adds the cipher suite to the list of Transport Layer Security (TLS) protocol cipher suites for the computer.
-If you do not specify a position in the list, this cmdlet adds it at the lowest position.
+If you do not specify a position in the list, this cmdlet adds it at the lowest position. No restart is required in order for changes to take effect.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/tls/Enable-TlsCipherSuite.md:25]:
```suggestion
If you do not specify a position in the list, this cmdlet adds it at the lowest position. No restart is required for changes to take effect.
```

File: docset/winserver2022-ps/tls/Enable-TlsCipherSuite.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ Enable-TlsCipherSuite [[-Position] <UInt32>] [-Name] <String> [-WhatIf] [-Confir
 ## DESCRIPTION
 The **Enable-TlsCipherSuite** cmdlet enables a cipher suite.
 This cmdlet adds the cipher suite to the list of Transport Layer Security (TLS) protocol cipher suites for the computer.
-If you do not specify a position in the list, this cmdlet adds it at the lowest position.
+If you do not specify a position in the list, this cmdlet adds it at the lowest position. No restart is required in order for changes to take effect.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/tls/Enable-TlsCipherSuite.md:25]:
```suggestion
If you do not specify a position in the list, this cmdlet adds it at the lowest position. No restart is required for changes to take effect.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Pull request merged**